cli/azd/pkg/auth/errors.go

```diff
@@ -62,13 +75,22 @@ func (e *ReLoginRequiredError) init(response *AadErrorResponse, scopes []string,
 		}
 	}
 
+	// The refresh token has expired or is invalid due to sign-in frequency checks by Conditional Access.
 	if slices.Contains(response.ErrorCodes, 70043) {
 		e.scenario = "login expired"
 	}
+
+	// User tried to sign in to a device from a platform not currently supported through Conditional Access policy
+	if slices.Contains((response.ErrorCodes), 50005) {
+		e.loginCmd += " --use-device-code=false"
+		// TODO: Use aka.ms short link
+		//nolint:lll
+		e.helpLink = "https://learn.microsoft.com/azure/developer/azure-developer-cli/troubleshoot#azd-pipeline-config-failure-due-to-conditional-access-policy"
+	}
 }
 
 func (e *ReLoginRequiredError) Error() string {
-	return fmt.Sprintf("%s, run `%s` to log in", e.scenario, e.loginCmd)
+	return e.description
```

Comment:
What do you think about naming this based on its intended usage, i.e. `errText` or `errString`?